Resources/Private/Extensions/solr/Templates/Search/HeaderTopSearch.html

```diff
@@ -0,0 +1,40 @@
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en"
```

Comment:
Would be good to update all class names for search to partly separate them from the header content element. So we will be able to use them in different header content elements, or even maybe on other content elements